dist/cookie/{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM crowdbotics/cb-django:3.8-slim-buster AS build
 
 # Copy dependency management files and install app packages to /.venv
 COPY backend/Pipfile backend/Pipfile.lock /
+COPY backend/modules/ modules/
```

Comment:
It should be COPY backend/modules/ /modules/, since the Pipfile is on /

Reply:
The `/` on line 4 is to points to the current folder, so adding a folder name without the `/` will create a new folder under the current folder.
`/modules/` and `modules/` will point to the same new folder, since the `modules` doesn't exist on the path `/` .

Reply:
Well, /modules/ and modules/ are same, in the context the current directory is /.

Reply:
Both ways work. I've tested it on [this](https://github.com/crowdbotics-dev/juan15022023-dev-63826) App
![imagen](https://user-images.githubusercontent.com/11747132/219443478-ef393cdc-4a0e-49a4-9f1a-b05f0ef81eae.png)
![imagen](https://user-images.githubusercontent.com/11747132/219443520-419c34d1-8ef7-4d7e-b635-7dd224488c9e.png)

![imagen](https://user-images.githubusercontent.com/11747132/219443587-eb480fca-1e8f-4a90-91bc-c483e230f2b6.png)
![imagen](https://user-images.githubusercontent.com/11747132/219443627-36ddbaae-4a09-44d3-8bc1-200ef1dbdbc8.png)

I've updated the path to start with `/` anyway.